### ISSUE TITLE
Cannot deploy Python project with standalone on Windows

### ISSUE THREAD

**Issue by kennedy** [serverless/serverless#7696]
I am following the getting started hello-world example using the `aws-python3` template, but I keep getting errors about:
```
Error: ENOENT: no such file or directory, copyfile 'C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\sdk-py\serverless_sdk\__init__.py'
```

Seems like I am not the only one. Here is a serverless forum post with no traction: [failure-to-deploy-or-run-locally-with-file-not-found-exception](https://forum.serverless.com/t/failure-to-deploy-or-run-locally-with-file-not-found-exception/10950)

No plugins were installed.

<details>
<summary><code>serverless.yml</code></summary>

```yaml
service: helloworld
app: helloworldtest
org: kennedy

provider:
  name: aws
  runtime: python3.8


functions:
  hello:
    handler: handler.hello
```

</details>

<details>
<summary><code>handler.py</code></summary>

```python
import json


def hello(event, context):
    body = {
        "message": "Go Serverless v1.0! Your function executed successfully!",
        "input": event
    }

    response = {
        "statusCode": 200,
        "body": json.dumps(body)
    }

    return response
```

</details>

<details>
<summary><b>CLI output</b></summary>

```
  Error --------------------------------------------------

  Error: ENOENT: no such file or directory, copyfile 'C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\sdk-py\serverless_sdk\__init__.py' -> 'C:\Users\Kennedy\Documents\projects\serverless\helloworld\serverless_sdk\__init__.py'
      at copyFileSync (fs.js:1807:3)
      at Object.copyFileSync (C:\snapshot\serverless\lib\utils\standalone-patch.js:53:5)
      at copyFile (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:68:8)
      at onFile (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:53:25)
      at getStats (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:48:44)
      at startCopy (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:38:10)
      at copyDirItem (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:120:10)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:113:39
      at Array.forEach (<anonymous>)
      at copyDir (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:113:23)
      at onDir (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:103:10)
      at getStats (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:45:37)
      at startCopy (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:38:10)
      at handleFilterAndCopy (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:33:10)
      at Object.copySync (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:26:10)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:313:10
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:8:103)
      at _next (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:10:194)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:10:364
      at new Promise (<anonymous>)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:10:97
      at wrap (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:387:17)
      at ServerlessEnterprisePlugin.<anonymous> (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\plugin.js:446:17)
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\plugin.js:3:103)
      at _next (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\plugin.js:5:194)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com

  Your Environment Information ---------------------------
     Operating System:          win32
     Node Version:              12.16.1
     Framework Version:         1.69.0 (standalone)
     Plugin Version:            3.6.8
     SDK Version:               2.3.0
     Components Version:        2.30.2

```

</details>

<b>Installed version</b>

```
Framework Core: 1.69.0 (standalone)
Plugin: 3.6.8
SDK: 2.3.0
Components: 2.30.2
```

**Comment by Micron-Flash** [comment]
I've been running into this same error as well. A workaround that seems to be working for me is to not enable the "You can monitor, troubleshoot. and test your new service with a free Serverless account" when setting up a new project

**Comment by johnthatfield** [comment]
I'm also having this issue. Exactly the same as described above

**Comment by medikoo** [comment]
@kennedy I understand that command is: `serverless invoke local --function hello` ? (description and CLI output doesn't state what command was invoked)

**Comment by johnthatfield** [comment]
Command is
`serverless deploy`

**Comment by medikoo** [comment]
@johnthatfield I do not reproduce this error for `serverless deploy`, but I see exactly same error as in description for `serverless invoke local --function hello` (and that was already reported at #7407 )

It might be that your case is about different error, if it's the please open other issue (but please respect all template remarks)

**Comment by medikoo** [comment]
Duplicate of #7407

**Comment by kennedy** [comment]
Forgive me, I do not understand. Is the solution not to use monitoring? Is this a known bug during deploy?

> 
> 
> @kennedy I understand that command is: `serverless invoke local --function hello` ? (description and CLI output doesn't state what command was invoked)

@medikoo the command I was running was `serverless deploy` just like the example/tutorial, I was presented with that error. 


When I attempt to `serverless deploy` with `org` and `app` configured in the `serverless.yaml`, it creates an empty directory called `serverless_sdk`. According to this [framework documentation](https://www.serverless.com/framework/docs/dashboard/sdk/python/) its a module I would import, but its empty.

---

> 
> 
> I've been running into this same error as well. A workaround that seems to be working for me is to not enable the "You can monitor, troubleshoot. and test your new service with a free Serverless account" when setting up a new project

@Micron-Flash I confirm I wont get this error if I don't use the free monitoring and dashboard. Though, I would like to use it. Seems like a bug some where.

---

> 
> 
> Duplicate of #7407

Unfortunately this situation is not the same. @adimango issue wasnt about deploys

**Comment by johnthatfield** [comment]
I second this @kennedy . Issue was closed for no reason. I too have the same bug on `serverless deploy`. Definitely needs investigating as the dashboard is completely useless with this bug

**Comment by medikoo** [comment]
@kennedy do you also observe it with latest version of a Framework? For me this service deploys without issues with v1.70.0 standalone.

If you observe same issue with v1.70.0. Please provide full output of `sls deploy` command (with `SLS_DEBUG=*` ensured).

Note that your report (1) missess the information on what command is used, and links to report on forum which is clearly about `sls invoke local`, (2) doesn't provide full CLI output, and (3) refers to an outdated version of a Framework.

@johnthatfield are you also on Windows?

**Comment by johnthatfield** [comment]
Yes, also on windows. Latest version downloaded thismorning

**Comment by medikoo** [comment]
I've reproduced it on Windows (it appears as specific to this OS)

**Comment by kennedy** [comment]
@medikoo Thanks for clarifying. The forum was talking about invoke, however it was the only time someone else documented the issue as me by [@devcoder1 on the forum on May 8, 2020](https://forum.serverless.com/t/failure-to-deploy-or-run-locally-with-file-not-found-exception/10950/2). 

As for what command I used, I apologize. I will remember to add the command next time, and I will update the issue and note the update; I thought my copy paste from my CLI included the command.

As for what version as of this writing:
```
Framework Core: 1.70.0 (standalone)
Plugin: 3.6.11
SDK: 2.3.0
Components: 2.30.10
```


<details>
<summary>Here is me rerunning on <code>1.70.0</code></summary>

```
C:\Users\Kennedy\Documents\projects\serverless\helloworld> serverless deploy
  Error --------------------------------------------------

  Error: ENOENT: no such file or directory, copyfile 'C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\sdk-py\serverless_sdk\__init__.py' -> 'C:\Users\Kennedy\Documents\projects\serverless\helloworld\serverless_sdk\__init__.py'
      at copyFileSync (fs.js:1807:3)
      at Object.copyFileSync (C:\snapshot\serverless\lib\utils\standalone-patch.js:53:5)
      at copyFile (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:68:8)
      at onFile (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:53:25)
      at getStats (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:48:44)
      at startCopy (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:38:10)
      at copyDirItem (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:120:10)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:113:39
      at Array.forEach (<anonymous>)
      at copyDir (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:113:23)
      at onDir (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:103:10)
      at getStats (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:45:37)
      at startCopy (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:38:10)
      at handleFilterAndCopy (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:33:10)
      at Object.copySync (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\node_modules\fs-extra\lib\copy-sync\copy-sync.js:26:10)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:308:10
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:8:103)
      at _next (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:10:194)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:10:364
      at new Promise (<anonymous>)
      at C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:10:97
      at wrap (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\wrap.js:382:17)
      at ServerlessEnterprisePlugin.<anonymous> (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\plugin.js:446:17)
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\plugin.js:3:103)
      at _next (C:\snapshot\serverless\node_modules\@serverless\enterprise-plugin\lib\plugin.js:5:194)
      at processTicksAndRejections (internal/process/task_queues.js:97:5)

     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.

  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com

  Your Environment Information ---------------------------
     Operating System:          win32
     Node Version:              12.16.1
     Framework Version:         1.70.0 (standalone)
     Plugin Version:            3.6.11
     SDK Version:               2.3.0
     Components Version:        2.30.10
```
</details>

**Comment by kennedy** [comment]
@medikoo As a fun addition, I was able to deploy without the `app`/`org` fields. But I am getting this error now when I do local invoke, might be a parsing issue?

```
python: can't open file 'C:\snap
shot\serverless\lib\plugins\aws\invokeLocal\invoke.py': [Errno 2] No such file or directory
```

**Comment by medikoo** [comment]
It's wierd that it attempts to access files needed for local invocation.

Did it work for you with older versions of standalone build?

**Comment by kennedy** [comment]
@medikoo The last project I worked on with serverless framework was back in September 2019, and standalone deploys worked.

**Comment by medikoo** [comment]
Will be fixed with #7699, and most likely published Monday morning. Great thanks for reporting!
